bedrock/mozorg/util.py

```diff
@@ -30,9 +30,7 @@ def page(name, tmpl, decorators=None, url_name=None, ftl_files=None, **kwargs):
     page's URL name will be "path.to.template". Set the `url_name` parameter
     to override this name.
 
-    @param name: The URL regex pattern.  If not empty, a trailing slash is
-        added automatically, so it shouldn't be included in the parameter
-        value.
+    @param name: The URL path. It is passed to `django.urls.path`.
```

Comment:
A test for a slash-less path would be good. Or even docs to say you can do something like `page("path/to/file.json", ...)`, if needed.

Reply:
Added test and a link to the Django docs in the docs.